src/odin/maneuver.cc

```diff
@@ -764,6 +766,36 @@ void Maneuver::set_contains_obvious_maneuver(bool contains_obvious_maneuver) {
   contains_obvious_maneuver_ = contains_obvious_maneuver;
 }
 
+bool Maneuver::has_combined_enter_exit_roundabout() const {
+  return has_combined_enter_exit_roundabout_;
+}
+
+void Maneuver::set_has_combined_enter_exit_roundabout(bool has_combined_enter_exit_roundabout) {
+  has_combined_enter_exit_roundabout_ = has_combined_enter_exit_roundabout;
+}
+
+float Maneuver::roundabout_length(const Options::Units& units) const {
+  if (units == Options::miles) {
+    return (roundabout_length_ * midgard::kMilePerKm);
+  }
+  return roundabout_length_;
+}
+
+void Maneuver::set_roundabout_length(float roundabout_length) {
```

Comment:
What units do we expect `roundabout_length` to be in? meters? Maybe add this info somewhere, either in comment or in the argument name.

Reply:
i will add comment